src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Interfaces/IApprovalsOuterApiClient.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ProviderCommitments.Interfaces
+{
+    public interface IApprovalsOuterApiClient
```

Comment:
Is there any value in creating this interface?  I think it's superfluous isn't it?

Reply:
No, but in the current setup but it does give us the possibility of injecting the new API client directly into the controller instead of going via the mediator

Reply:
If it's not useful at the moment I don't think we should add the interface.  We can always add it in the future when if we do need it.  Right now it's just another layer of complication that's unnecessary.

Reply:
Looking at the way this app works, it generally uses mappers to produce the view models and not the Application/Query patterns to produce the view models. We should continue in the same pattern I think